### PR TITLE
docs(sensor calibration): add lidar imu calibration tool

### PR DESCRIPTION
**Pull request by JianKangEgon**
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/sensor-calibration.md [hunk 1]
```diff
@@ -15,3 +15,9 @@ TBD
 ## Lidar-camera calibration
 
 TBD
+
+## Lidar-IMU calibration
+
+This calibration tool, developed by [APRIL Lab](https://github.com/APRIL-ZJU) in Zhejiang University in China, is a toolkit for calibrating the 6DoF rigid transformation and the time offset between a 3D LiDAR and an IMU. It's based on continuous-time batch optimization. IMU-based cost and LiDAR point-to-surfel distance are minimized jointly, which renders the calibration problem well-constrained in general scenarios.
+
+[Autocore](https://autocore.ai/) has overwritten the Lidar input for a more general usage. For more details including instructions, trouble shooting or rosbag etc., please refer to the link [here](https://gitlab.com/JianKang_Egon/li_calib)
```
Comment (LalithVipulananthan) [inline docs/how-to-guides/sensor-calibration.md:23]:
```suggestion
[Autocore](https://autocore.ai/) has overwritten the Lidar input for a more general usage. For more details including instructions, trouble shooting or rosbag etc., please refer to the link [here](https://gitlab.com/JianKang_Egon/li_calib)
```
```suggestion
[Autocore](https://autocore.ai/) has forked the original LI-Calib tool and overwritten the Lidar input for more general usage. Information on how to use the tool, troubleshooting tips and example rosbags can be found at the [LI-Calib fork on GitLab](https://gitlab.com/JianKang_Egon/li_calib).
```

**Review by LalithVipulananthan — COMMENTED** [review]
Minor corrections suggested, but otherwise looks fine.

File: docs/how-to-guides/sensor-calibration.md [hunk 2]
```diff
@@ -15,3 +15,9 @@ TBD
 ## Lidar-camera calibration
 
 TBD
+
+## Lidar-IMU calibration
+
+This calibration tool, developed by [APRIL Lab](https://github.com/APRIL-ZJU) in Zhejiang University in China, is a toolkit for calibrating the 6DoF rigid transformation and the time offset between a 3D LiDAR and an IMU. It's based on continuous-time batch optimization. IMU-based cost and LiDAR point-to-surfel distance are minimized jointly, which renders the calibration problem well-constrained in general scenarios.
```
[the remark this answers is not in the document]
Comment (kenji-miyake) [reply docs/how-to-guides/sensor-calibration.md:21]: I think it's okay to add it to the dictionary, but it's better to explain the word briefly here. I didn't know the word.

**Comment by LalithVipulananthan** [comment]
@JianKangEgon Hi there. You seem to have added a lot of files to this commit that are unrelated to the sensor calibration document. Would it be possible to revert those commits and create a separate PR (and a new branch) for those changes please?

**Comment by JianKangEgon** [comment]
@LalithVipulananthan Sorry, something went wrong after sign-off and I don't know how to correct it. I'll close this pr and propose a new one to make it

**Pull request closed without merging**